VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/config/E2EConfig.java

```diff
@@ -0,0 +1,19 @@
+package org.edx.mobile.test.feature.config;
+
+import android.support.annotation.NonNull;
+import android.text.TextUtils;
+
+import org.edx.mobile.util.Config;
+
+import java.util.UUID;
+
+public class E2EConfig {
```

Comment:
It seems needlessly confusing to have two different ``EndToEndConfig`` classes and isn't how any of our other config parts work. Probably we can just move these constants into the other class?